DuckDuckGo/Tab/Model/SystemInfo.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  SystemInfo.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Common
+
+#if !APPSTORE
+
+final class SystemInfo {
+
+    static func pixelParameters(appVersion: AppVersion = AppVersion.shared) -> [String: String] {
+        let availableMemoryPercent = Self.getAvailableMemoryPercent()
+        let availableDiskSpacePercent = Self.getAvailableDiskSpacePercent()
+        return [
+           "available_memory": String(format: "%.2f", availableMemoryPercent),
```

Comment:
Weirdly enough this is always formatting my available memory as `0.00%`. Using `String(format: "%.2f", Double(availableMemoryPercent))` fixes it, but also available memory is always reported rounded to full percent, so perhaps we should use `String(format: "%d", availableMemoryPercent)` or just `String(availableMemoryPercent)`?

Reply:
Yeah, interpolation is probably fine for that one.  Thanks.